packages/dnb-eufemia/src/extensions/forms/Field/Expiry/__tests__/Expiry.test.tsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react'
+import { act, render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import * as Field from '../..'
+
+describe('Field.Expiry', () => {
+  beforeEach(() => {
+    window.requestAnimationFrame = jest.fn((callback) => {
+      return setTimeout(callback, 0)
+    })
+    window.cancelAnimationFrame = jest.fn((id) => {
+      clearTimeout(id)
+      return id
+    })
+  })
+
+  it('should return month and year values as a concacted string', async () => {
+    const onChange = jest.fn()
+
+    render(<Field.Expiry onChange={onChange} />)
+
+    const input = document.querySelector('input')
+
+    act(() => {
+      input.focus()
+    })
+
+    await userEvent.keyboard('1235')
+
+    expect(onChange).toBeCalledTimes(4)
+    expect(onChange.mock.calls[0][0]).toEqual('1')
+    expect(onChange.mock.calls[1][0]).toEqual('12')
+    expect(onChange.mock.calls[2][0]).toEqual('123')
+    expect(onChange.mock.calls[3][0]).toEqual('1235')
```

Comment:
```suggestion
    expect(on_blur).toHaveBeenLastCalledWith('1235')
```

When using "Last", you can remove all the other checks. Should be fine enough. You already did check for `toBeCalledTimes`.